### PR TITLE
tiltfile: print output of local() to the logger

### PR DESCRIPTION
**Pull request by nicks**
Hello @jazzdan, @yuindustries,

Please review the following commits I made in branch nicks/running:

4799f142e3dbcf2fd64ab677a119e2f5a7cb839a (2019-07-09 15:02:18 -0400)
tiltfile: print output of local() to the logger

### CODE REVIEW

**Comment by nicks** [comment]
Screenshot from 2019-07-09 15-01-47

**Comment by nicks** [comment]
i was trying to debug some problems and having a lot of trouble...this seems generally more useful

File: internal/tiltfile/docker.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ type dockerImage struct {
 	baseDockerfile     dockerfile.Dockerfile
 	configurationRef   container.RefSelector
 	deploymentRef      reference.Named
-	syncs              []sync
+	syncs              []pathSync
```
Comment (jazzdan) [inline internal/tiltfile/docker.go]: Are these changes supposed to be here?

Comment (nicks) [reply internal/tiltfile/docker.go]: yes. you can't have an `import` and a `type` with the same name. the go compiler yells at you

Comment (jazzdan) [reply internal/tiltfile/docker.go]: Ah, gotcha makes sense

**Pull request merged**